corehq/apps/reports/filters/users.py

```diff
@@ -313,8 +311,8 @@ def _selected_location_entries(self, mobile_user_and_group_slugs):
         location_ids = self.selected_location_ids(mobile_user_and_group_slugs)
         if not location_ids:
             return []
-        return map(self.utils.location_tuple,
-                   SQLLocation.objects.filter(location_id__in=location_ids))
+        return list(map(self.utils.location_tuple,
+                        SQLLocation.objects.filter(location_id__in=location_ids)))
```

Comment:
why was this change needed?

Reply:
in python3, `map` returns an iterable, so this makes sure that it's a list.